src/qt/rpcconsole.cpp

```diff
@@ -274,6 +279,25 @@ RPCConsole::RPCConsole(const PlatformStyle *platformStyle, QWidget *parent) :
     connect(ui->fontSmallerButton, SIGNAL(clicked()), this, SLOT(fontSmaller()));
     connect(ui->btnClearTrafficGraph, SIGNAL(clicked()), ui->trafficGraph, SLOT(clear()));
 
+    //Setup autocomplete and attach it
+    QStringList wordList;
+    for (int i =0; i < 53; ++i)
+    {
+        wordList<<vRPCCommands[i].name.c_str();
+    }
+    #ifdef ENABLE_WALLET
+    bool fDisableWallet = GetBoolArg("-disablewallet", false);
+    if(!fDisableWallet)
+    {//Only add commands if wallet is not disabled
```

Comment:
No need for this comment. Also you don't have to parse the arg. Maybe try just `if (pwalletMain)`?